sycl/test/backend/cuda/primary_context.cpp

```diff
@@ -0,0 +1,82 @@
+// REQUIRES: cuda
+// RUN: %clangxx %s -I%opencl_include_dir -I%cuda_toolkit_include -o %t.out -lcuda -lsycl
```

Comment:
Why do you need `-I%opencl_include_dir` in this test?

Reply:
Without it, the OpenCL header supplied with the CUDA toolkit is picked up which causes problems as it is behind OpenCL 2.1

Reply:
Oh, it seems to be needed for the SYCL runtime, not for the test

I think it worth to put this comment into directly into the test - someone not familiar with the issue might step into it while adding new tests

Reply:
Good idea. I've added a note about this.